cmake/common.cmake

```diff
@@ -266,6 +266,85 @@ function(add_perf_test)
   )
 endfunction()
 
+# Helper for building end-to-end perf tests using the python infrastucture
+function(add_piccolo_test)
```

Comment:
We've been dragging around unused arguments for a while now, to stay compatible with the old perftest binary. Since we're re-shuffling the CLI in this PR anyway, might as well split.